modules/lite/src/deckgl-json.js

```diff
@@ -0,0 +1,31 @@
+import DeckGL from './deckgl.js';
```

Comment:
Why add it to the standalone module?

Reply:
> Why add it to the standalone module?

Good catch, it was part of the initial experimental code, will remove. 

My proposal is to add it to a separate `json` module, I will update code accordingly.